planning/behavior_path_planner/include/behavior_path_planner/behavior_path_planner_node.hpp

```diff
@@ -113,12 +129,14 @@ class BehaviorPathPlannerNode : public rclcpp::Node
   void onForceApproval(const PathChangeModule::ConstSharedPtr msg);
   void onMap(const HADMapBin::ConstSharedPtr map_msg);
   void onRoute(const HADMapRoute::ConstSharedPtr route_msg);
+  SetParametersResult paramCallback(const std::vector<rclcpp::Parameter> & parameters);
```

Comment:
[imo]
It is better to unify naming rule for callback function. (e.g. `onParam`)
What do you think ?

Reply:
Thank you very much!
Yes. That is true. I've renamed them to `onSetParam` instead.